eng/performance/scenarios.proj

```diff
@@ -75,6 +75,14 @@
       <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
     </HelixWorkItem>
   </ItemGroup>
+
+  <!-- Blazor 3.2 Template SOD -->
+  <ItemGroup>
+    <HelixWorkItem Include="Blazor 3.2 Template SOD" Contidion="'$(_Framework)' == 'netcoreapp3.1'">
+      <PreCommands>$(Python) pre.py --msbuild-static AdditionalMonoLinkerOptions=&quot;$(AdditionalMonoLinkerOptions) --dump-dependencies&quot;</PreCommands>
+      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
+    </HelixWorkItem>
+  </ItemGroup>
```

Comment:
Are you also automating installing the Blazor templates?

Reply:
Oops missed that.